main.go

```diff
@@ -161,6 +161,9 @@ func updatesJSON(rs []*moduleReference) {
 	}
 	go func() {
 		for o := range out {
+			if o.Version == o.LatestWithoutConstraint {
```

Comment:
generally agree on being able to skip update-less modules in the output. We should still have a way to print _all_ of them though. Since the features are adding up, I think we will need to have a subcommand style CLI to keep all these options manageable.

Reply:
Okay, so subcommand style CLI means we have options for commands too?
So do you have something in mind or I should remove the added checks in both `json` style and `pretty` style and get this PR updated?